File: Challenge1/JavaScript/Challenge1.js
function MultipleSums(firstNumber, secondNumber, total) {
    var totalNumber = 0;
    for (i = 0; i < total; i++) {
        if (i % firstNumber == 0) {
            totalNumber += i;
        }
        if (i % secondNumber == 0 && i % firstNumber != 0) {
            totalNumber += i;
        }
    }
    return totalNumber;
}

console.log(MultipleSums(3, 5, 1000));

File: Challenge1/Java/Challenge1.java
public class Challenge1{

    public static void main(String[] args){
        System.out.println(MultipleSums(3, 5, 1000));
    }

    public static int MultipleSums(int firstNumber, int secondNumber, int total)
    {
        int totalNumber = 0;
        for (int i = 0; i < total; i++)
        {
            if (i % firstNumber == 0 || i % secondNumber == 0)
            {
                totalNumber += i;
            }
        }
        return totalNumber;
    }
}
